.github/workflows/js_tests.yml

```
@@ -12,26 +12,10 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  lint_custom:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version: [14]
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup Node
-        uses: actions/setup-node@v1
-        with:
-          node-version:  ${{ matrix.node-version }}
-      - name: Npm install
-        run: |
-          npm install
-      - name: Run custom plugin linter
-        run: |
-          npm run lint:custom
   test:
     name: JavaScript
     uses: theforeman/actions/.github/workflows/foreman_plugin_js.yml@v0
     with:
       plugin: foreman_openscap
+      foreman_version: gh-actions
+      foreman_repository: MariaAga/foreman
```

Maybe this will work? I mean, there is almost no difference, I just always used this:
```suggestion
      foreman_version: refs/pull/10572/head
```

---

It does look more like what I saw in the past, but couldnt find it written anywhere 
Thanks